airflow/api_fastapi/core_api/routes/public/__init__.py

```diff
@@ -41,24 +41,22 @@
 public_router = AirflowRouter(prefix="/public")
 
 
+public_router.include_router(assets_router)
 public_router.include_router(backfills_router)
-public_router.include_router(dags_router)
 public_router.include_router(connections_router)
 public_router.include_router(dag_run_router)
 public_router.include_router(dag_sources_router)
+public_router.include_router(dag_stats_router)
+public_router.include_router(dag_warning_router)
 public_router.include_router(dags_router)
 public_router.include_router(event_logs_router)
 public_router.include_router(import_error_router)
 public_router.include_router(monitor_router)
-public_router.include_router(dag_warning_router)
 public_router.include_router(plugins_router)
 public_router.include_router(pools_router)
 public_router.include_router(providers_router)
 public_router.include_router(task_instances_router)
-public_router.include_router(variables_router)
+public_router.include_router(tasks_router)
 public_router.include_router(variables_router)
 public_router.include_router(version_router)
-public_router.include_router(dag_stats_router)
 public_router.include_router(xcom_router)
```

Comment:
Also made them alphabetically ordereded -- not strictly needed but good for my eyes :D